src/billing/List/BillingList.tsx

```diff
@@ -291,6 +307,27 @@ export const BillingList = (props: BillingListProps) => {
           <h2 style={{ fontSize: 16 }}>Billing Projects</h2>
           <CreateBillingProjectControl showCreateProjectModal={showCreateProjectModal} />
         </div>
+        {isFeaturePreviewEnabled(CONSOLIDATED_SPEND_REPORT) && (
+          <Clickable
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              color: colors.accent(1.1),
+              ...(type === 'consolidatedSpendReport' ? { backgroundColor: colors.dark(0.1) } : {}),
+              ...(type === 'consolidatedSpendReport' ? { fontWeight: 'bold' } : {}),
+              marginLeft: '2rem',
+              marginRight: '2rem',
+              padding: '.5rem .5rem .5rem',
+            }}
+            href={`${Nav.getLink('billing')}?${qs.stringify({
+              type: 'consolidatedSpendReport',
+            })}`}
+            aria-current={false}
+            onClick={void Metrics().captureEvent(Events.billingViewConsolidatedSpendReport)}
```

Comment:
The mixpanel event is captured only when the user clicks on the link, so doesn't count things like logging out and then logging back in still on this page or refreshing.  I think that should be correct, since there is currently no other way to get to this page, but if there ever appears another link here then we might want to update the event to happen on loading the page.